standard/sections/clause_7_normative_text.adoc

```diff
@@ -1,128 +1,119 @@
-== The WIS2 Notification Message Encoding
+== WIS2 Notification Message
+Note: This section of working draft document is the same as Appendix E in the https://library.wmo.int/idurl/4/68731[_Manual on the WMO Information System_] (WMO-No. 1060), Volume II
 
-A WIS2 Notification Message (WNM) provides descriptive information about data or metadata made available through WIS 2.0. The standard notification message format ensures that the WIS2 ecosystem (data publisher, data user, and global services) is a robust, effective, and unified data exchange platform for weather, climate, and water.
+The WIS2 Notification Message (WNM) is an extension of the OGC API - Features standard  and shall be encoded in GeoJSON. The normative provisions in this standard are denoted by the base URI (``http://wis.wmo.int/spec/wnm/1``) and requirements are denoted by partial URIs relative to this base. Property names, values and examples are represented with ``shaded`` text in this document.
 
-WIS2 Notification Message documents are provided as MQP payloads by WIS2 nodes, Global Broker services, as well as Replay API services.
-
-A WNM provides notification metadata about data or metadata published to WIS2, including when the data was published, its spatial and temporal characteristics, and where to access and download.
-
-WIS2 Notification Message is an extension of the OGC API - Features standard [3] and shall be encoded in GeoJSON (RFC7946 [4]).
-
-=== Requirements Class "Core"
-
-==== Overview
-
-This Core Requirements Class provides requirements for a WIS2 Notification Message.
+:sectnums!:
+=== 1. Requirements Class "Core"
 
 include::../requirements/requirements_class_core.adoc[]
 
+==== 1.1 Overview
+
 The table below provides an overview of the set of properties that may be included in a WNM.
 
 .WNM core properties
 |===
 |Property|Requirement|Description
 
 |``id``
-|**required**
-|A universally unique identifier of the message (see <<_identifier>>)
-
+|**Required**
+|A universally unique identifier (UUID) of the message (see <<_1_4_identifier>>)
 |``type``
-|**required**
-|A fixed value denoting the record as a GeoJSON ``Feature`` (see <<_geojson_compliance>>)
+|**Required**
+|A fixed value denoting the record as a GeoJSON ``Feature`` (see <<_1_3_geojson_compliance>>)
 
 
 |``conformsTo``
-|**required** (one of ``conformsTo`` or ``version``)
-|The version of WNM associated that the record conforms to (see <<_conformance_2>>.
+|**Required** (one of ``conformsTo`` or ``version``)
```

Comment:
to discuss

Reply:
leave in PR branch, but don't include publication doc because it is a FT amendment.